### DISCUSSION TITLE
Open an external file from GAMA

### DISCUSSION THREAD

**Discussion by hana-gharrad** [gama-platform/gama#61] [Q&A] [answered]
Hello,

I'm actually trying to run matlab evaluation example from GAMA. I pointed the correct folder for the activate matlab but it always shows "file not found". I tried a normal file reading from absolute path it shows the same error. My question, what should I set as path_to_matlab in windows system.
string path_to_matlab <- "C:/matlab/..../bin/win64/";   
Also, is it possible to use absolute path for files:
file my_file <- csv_file("C:/Users:/....../test.csv"

Many thanks,

**Comment by benoitgaudou** [comment]
Dear,

Just to let you know, there is an issue on the Matlab plugin, mainly due to
the internal loading of the Matlab library.
In addition, the error is not really clear: the file not found error you
getif mainly due to an issue in this loading.

We need to update the plugin to change that and review the documentation
explaining the loading of Matlab.

Cheers

Benoit

Le dim. 2 avr. 2023 à 11:10, hana-gharrad ***@***.***> a
écrit :

**Reply by hana-gharrad** [reply]
Hi Benoit,

Thank you for the clarification. I saw the version 1.9.0 has been released, is the matlab library load is fixed in the released version?

Kind regards,

**Comment by benoitgaudou** [comment]
Dear,

Not yet. I planned to work on it during the end of the week.
Cheers

Benoit

Le jeu. 13 avr. 2023 à 06:15, hana-gharrad ***@***.***> a
écrit :

**Comment by benoitgaudou** [comment, marked as answer]
Dear,
I updated the plugin to connect to Matlab.
Up to now it can only be tested with GAMA git version.
It has been created to work on Windows (64 bits) with the last release of
Matlab.
(
https://github.com/gama-platform/gama.experimental/tree/GAMA_1.9.0/ummisco.gama.extensions.matlab#readme
)

Cheers

Benoit

Le jeu. 13 avr. 2023 à 11:25, Benoit Gaudou ***@***.***> a
écrit :

**Reply by hana-gharrad** [reply]
Dear Benoit,

It work like a charm with Matlab23a in Windows. Thank you so much for you help.

Kind regards,
